libs/kano-profile/src/kano_profile_apps.cpp

```diff
@@ -53,3 +104,17 @@ long kano_profile::apps::load_app_state_variable(std::string app, std::string va
 
     return var;
 }
+
+std::string kano_profile::apps::load_app_state_variable_encode(
+    std::string app, std::string variable) const
+{
+    PyObject *py_var = this->run_func(
+        "load_app_state_variable_encode", new_tuple(app, variable)
+    );
+
+    std::string var = (py_var == NULL) ? "" : PyString_AsString(py_var);
```

Comment:
Should probably play this safe and check that the python type is a string before decoding it with [`PyString_Check(py_var)`](https://docs.python.org/2/c-api/string.html#c.PyString_Check) as [`PyString_AsString()`](https://docs.python.org/2/c-api/string.html#c.PyString_AsString) raises an exception in this case.